Dangerfile

```diff
@@ -3,8 +3,8 @@
 require 'ostruct'
 
 module VSPDanger
-  HEAD_SHA = `git rev-parse --abbrev-ref HEAD`.chomp.freeze
-  BASE_SHA = 'origin/master'
+  HEAD_SHA = ENV.fetch('GITHUB_HEAD_REF') ? "origin/#{ENV.fetch('GITHUB_HEAD_REF')}" : `git rev-parse --abbrev-ref HEAD`.chomp.freeze
```

Comment:
Do you have to check ENV.fetch('GITHUB_HEAD_REF') for empty/nil?

Reply:
@LindseySaari I did that because if those vars ARE nil, it'll evaluate to `"origin/"`, but I'd rather it default to `'git rev-parse --abbrev-ref HEAD'.chomp.freeze`/`origin/master` in the case it is being ran outside of the GitHUB CI context (I believe it can be ran locally, per our convo yesterday).